tests/suite/test_virtual_server_dos.py

```diff
@@ -206,6 +215,12 @@ def test_dos_vs_logs(
         print(f"log_loc: {log_loc} syslog_pod: {syslog_pod} namespace: {ingress_controller_prerequisites.namespace}")
         log_contents = get_file_contents(kube_apis.v1, log_loc, syslog_pod, ingress_controller_prerequisites.namespace)
 
+        # print("\n===================== ADMD Logs Start =====================")
```

Comment:
@pasmant can you please remove the commented code if not needed?

Reply:
There is any log level in pytest?

Reply:
Remove the logs